rst_files/testing.rst

```diff
@@ -254,21 +257,22 @@ For now, let's just try adding a dependency
 
 1. Activate the package environment (to be run from the base, ``v1.0`` environment)
 
-.. code-block:: julia
+  .. code-block:: julia
 
-    ] activate ExamplePackage
+      ] activate ExamplePackage
 
-This tells Julia to write the results of package operations to ``ExampleProject``'s TOML, and use the versions of packages specified there 
+  This tells Julia to write the results of package operations to ``ExampleProject``'s TOML, 
+  and use the versions of packages specified there 
 
 2. Add a package
 
-.. code-block:: julia
+  .. code-block:: julia
 
-    ] add Expectations
+      ] add Expectations
 
-We can track changes in the TOML, as before
+  We can track changes in the TOML, as before
 
-Here's the Manifest
+  Here's the manifest
```

Comment:
What is a "Manifest"? Should this be capitalised? Could we define it?

Reply:
Yeah, it refers to the `Manifest.toml` file. Will clarify.